Automattic/sensei
# Add showcase courses upsell

Resolves #6645
Pair with https://github.com/Automattic/sensei-pro/pull/2197

## Proposed Changes
* It adds the showcase courses upsell, in the same place where we add the link in Sensei Pro.

## Testing Instructions


1. With Sensei Pro deactivated, go to the courses in WP Admin, and make sure you see the link "Showcase Courses".
2. Navigate to the "Showcase Courses", and make sure you see the upsell.
3. Checkout the branch from https://github.com/Automattic/sensei-pro/pull/2197 in Sensei Pro.
4. Activate Sensei Pro.
5. Navigate to "Showcase Courses", and make sure you see the Showcase list.

## Pre-Merge Checklist


## Screenshots

<img width="1490" alt="Screenshot 2023-03-16 at 11 17 08" src="https://user-images.githubusercontent.com/876340/225645518-1d2df7ab-445d-4e0b-9e26-7f52d20e48bc.png">
<img width="497" alt="Screenshot 2023-03-16 at 11 17 20" src="https://user-images.githubusercontent.com/876340/225645554-7a3ddbf0-5b3c-4f2e-a141-77537590cfab.png">
